docker/dev-env/Dockerfile

```diff
@@ -51,7 +50,7 @@ RUN apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
 
-COPY --from=opensearchproject/opensearch:1.3.11 --chown=$USER_NAME:$USER_GROUP /usr/share/opensearch /usr/share/opensearch
+COPY --from=opensearchproject/opensearch:1 --chown=$USER_NAME:$USER_GROUP /usr/share/opensearch /usr/share/opensearch
```

Comment:
no danger in unpinning specific sub-version(s) here?  They're probably not going to do much more development on it, but this holds us on v1 but is there any concern with changes between v1.3.11 -> v1.3.15?

Reply:
No, I don't think so. If something breaks here with a point version, I would be glad to know of it.

Reply:
@sfreudenthaler Yeah this links in with what I just posted in slack about reproducible builds.   I understand the difficulty though as there are problems if the version is pinned also.   If you do not have a good strategy including things like dependabot, or other processes to check and update versions developers will tend to not want to touch anything which means you end up falling behind on changes. (Maven has a nice command to tell you what can be updated, but you have to use it.   If we let it upgrade automatically think it keeps us nicely in line with any changes, but when there is an issue it can be hard to diagnose and can end up randomly directly in master or release branches.

Reply:
This is also made more difficult if the same version number is used accross the repository requiring a search and replace to be consistent,   it is better to parameterize and have one location the the value e.g. in parent/pom.xml that is referenced everywhere that needs it.  
Not sayign we have to change anything to this PR, but we need to consider our update strategy for external dependencies to the build.